po/pl.po

```diff
@@ -4,8 +4,8 @@ msgstr ""
 "Report-Msgid-Bugs-To: https://bugs.gnucash.org/enter_bug.cgi?"
 "product=GnuCash&component=Translations\n"
 "POT-Creation-Date: 2019-12-26 16:42+0100\n"
```

Comment:
On which git version is your work based? cstim applied a few changes in recent days.

Reply:
The latest one, which contains cstim's changes.